sample-apps/delivery-customizations-sample-app/web/frontend/components/CustomizationForm.jsx

```diff
@@ -0,0 +1,50 @@
+import { Form, FormLayout, TextField, Button, Badge } from "@shopify/polaris";
+
+export function CustomizationForm({
+  deliveryOptionName,
+  enabled,
+  operation,
+  isNewCustomization = false,
+  onSubmit,
+  onInputChange,
+  loading,
+  disabled,
+  ...props
+}) {
+  const handleSubmit = () => {
+    if (onSubmit) onSubmit({ deliveryOptionName, enabled, operation });
+  };
+
+  const handleInputChange = (value, name) => {
+    if (onInputChange) onInputChange({ value, name });
+  };
+
+  return (
+    <Form onSubmit={handleSubmit} {...props}>
+      <FormLayout>
+        <FormLayout.Group>
+          {enabled || isNewCustomization ? (
+            <Badge status="success">Active</Badge>
+          ) : (
+            <Badge>Inactive</Badge>
+          )}
+        </FormLayout.Group>
+        <FormLayout.Group>
+          <TextField
+            type="text"
+            label="Delivery Option Name"
+            helpText="Will match the delivery option name exactly."
```

Comment:
I wonder if we can make this more descriptive of what the function does. Maybe something like: "Any delivery option matching this name exactly will be hidden." for hiding.

Otherwise, I think we should just remove this because the text is a little confusing IMO.